nginx/config.go

```diff
@@ -201,7 +201,7 @@ http {
 			proxy_set_header Upgrade $http_upgrade;
 			proxy_set_header Connection $connection_upgrade;
 
-			{{ if $enforceHTTPS }}if ($access_scheme != "https") {
+			{{ if or $enforceHTTPS $appConfig.SSL.Enforce }}if ($access_scheme != "https") {
```

Comment:
Doesn't this mean that redirect can't be disabled on a per app basis, if it is enabled globally?